src/edu/washington/escience/myria/api/DatasetResource.java

```diff
@@ -555,6 +557,39 @@ private Response doIngest(
   }
 
   /**
+   * Ingests a dataset from S3 in parallel
+   *
+   * @param dataset the dataset to be ingested.
+   * @return the created dataset resource.
+   * @throws DbException if there is an error in the database.
+   * @throws InterruptedException
+   * @throws URIException
+   */
+  @POST
+  @Path("/ParallelIngest")
```

Comment:
Looks like we're using camelCase verbs for other methods, so should be `parallelIngest`.